Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -380,6 +380,8 @@ @implementation WKFileUploadPanel {
     RetainPtr<NSSet<NSString *>> _acceptedUTIs;
     OptionSet<WKFileUploadPanelImagePickerType> _allowedImagePickerTypes;
     CGPoint _interactionPoint;
+    CGPoint _interactionPointInWindow;
```

Comment:
I don't think this instance variable is necessary? Why not just compute it where it's used?

Reply:
That's how it was before. It's not computed where it's used anymore b/c of _timing issue_. If I'm correct, the menu would be/shown at the bottom of the web view. The view is then pushed up by the keyboard and the menu position in relation to the _window_ would now be right above the keyboard (even though it's not shown there). Our checks will then say that nothing is obscuring the menu b/c the position is _above_ the keyboard.

Reply:
Sorry, I got confused after seeing `webView` variable 😅

Makes sense since the content view is resized!